### PR TITLE
[2016.3] Merge forward from 2015.8 to 2016.3

### PR DESCRIPTION
**Pull request by rallytime**
Conflicts:
- doc/ref/configuration/minion.rst

### CODE REVIEW

File: salt/scripts.py [hunk 1]
```diff
@@ -107,11 +107,6 @@ def salt_minion():
     if '' in sys.path:
         sys.path.remove('')
 
-    if salt.utils.is_windows():
-        minion = salt.cli.daemons.Minion()
-        minion.start()
-        return
-
     if '--disable-keepalive' in sys.argv:
```
Comment (thatch45) [inline salt/scripts.py]: @twangboy did we not want this merged forward into 2016.3?

Comment (rallytime) [inline salt/scripts.py]: Ah right, probably not.

Comment (twangboy) [inline salt/scripts.py]: @rallytime Please do not include this one

Comment (rallytime) [inline salt/scripts.py]: Fixed.

Comment (twangboy) [inline salt/scripts.py]: @rallytime Did you remove the `if salt.utils.is_windows()` block? It needs to stay in there for 2016.3

Comment (twangboy) [inline salt/scripts.py]: @rallytime Sorry, I see it now...

Comment (rallytime) [inline salt/scripts.py]: @twangboy I put it back right here: https://github.com/saltstack/salt/pull/33322/commits/d66c9929183965bd593a156fc4a35100ab3ae8d9

**Pull request merged**